src/api/schema.ts

```diff
@@ -299,7 +299,8 @@ export const lists = {
       financialUk: relationship({ ref: 'LossPayeeFinancialUk.lossPayee' }),
       financialInternational: relationship({ ref: 'LossPayeeFinancialInternational.lossPayee' }),
       isAppointed: nullableCheckbox(),
-      locatedInUk: nullableCheckbox(),
+      isLocatedInUk: nullableCheckbox(),
```

Comment:
are these being tested in keystone.test.ts?

Reply:
No, this part is created in the `createAnApplication` helper instead of via the keystone "hooks" config (need to clean this up)